internal/v1/handler_blueprints.go

```diff
@@ -132,6 +132,31 @@ func (h *Handlers) GetBlueprint(ctx echo.Context, id openapi_types.UUID) error {
 	return ctx.JSON(http.StatusOK, blueprintResponse)
 }
 
+func (h *Handlers) GetImageTypesFromBlueprint(ctx echo.Context, id openapi_types.UUID) error {
+	userID, err := h.server.getIdentity(ctx)
+	if err != nil {
+		return err
+	}
+
+	ctx.Logger().Infof("Fetching blueprint %s", id)
+	blueprintEntry, err := h.server.db.GetBlueprint(ctx.Request().Context(), id, userID.OrgID())
+	if err != nil {
+		return err
+	}
+
+	blueprint, err := BlueprintFromEntry(blueprintEntry)
+	if err != nil {
+		return err
+	}
+
+	var imageTypes []ImageTypes
```

Comment:
Just a nitpick, you do not have to rebase this here we only have 3 elements but it is good to do this if you know the length in advance:

```suggestion
	imageTypes := make([]ImageTypes, 0, len(blueprint))
```